sdk/core/azure-core/azure/core/pipeline/policies/_universal.py

```diff
@@ -280,27 +281,31 @@ def on_request(self, request):
                 return
 
             try:
-                _LOGGER.debug("Request URL: %r", http_request.url)
-                _LOGGER.debug("Request method: %r", http_request.method)
-                _LOGGER.debug("Request headers:")
+                log_string = "Request URL: '{}'".format(http_request.url)
+                log_string += "/nRequest method: '{}'".format(http_request.method)
```

Comment:
Shouldn't `\n` be used? `/n` makes the output unreadable:

![image](https://user-images.githubusercontent.com/4003950/156154578-2c5e9a1e-2ee5-4bb1-9046-3401bace571a.png)